pinot-connectors/pinot-spark-2-connector/pom.xml

```diff
@@ -35,6 +35,45 @@
   </properties>
 
   <profiles>
+    <profile>
+      <id>build-shaded-jar</id>
+      <activation>
+        <activeByDefault>true</activeByDefault>
+      </activation>
+      <build>
+        <plugins>
+          <plugin>
+            <artifactId>maven-shade-plugin</artifactId>
```

Comment:
Previously the plugin was configured in the scala 2.12 profile, which I think it is incorrect. Specifically in master when using scala 2.13 some packages are not be shaded (or not in the same way).